packages/lib/src/components/TextInputWithGlyph/index.tsx

```diff
@@ -27,6 +34,23 @@ const TextInputWrapper = styled.div`
   width: 100%;
```

Comment:
It could be worth adding some padding to prevent the icon overlap

<img width="858" alt="Screenshot 2025-06-13 at 10 10 54 AM" src="https://github.com/user-attachments/assets/168e77d7-7868-4fb1-9d43-2e79fc5e506c" />


```suggestion
  width: 100%;
  
  input {
    padding-right: ${size.m};
  }
```